packages/util-dev/src/mockProviders/mockHandleProvider.ts

```diff
@@ -0,0 +1,11 @@
+import { Cardano } from '@cardano-sdk/core';
+
+export const Handle = {
+  handle: 'AdaHandle',
```

Comment:
```suggestion
  handle: 'mary',
```

Reply:
e25a4e19c0746a361b37 this should solve this